src/controllers/graphql/resolvers/reply/replyResolver.js

```diff
@@ -1,12 +1,66 @@
 const db = require('../../connectors');
 const Reply = db.sequelize.models.reply;
+const ReplyService = require('../../../../services/replyService');
+const {
+  AuthenticationError,
+  gql,
+  makeExecutableSchema
+} = require('apollo-server');
 
-module.exports = {
+const typeDefs = gql`
+  scalar Date
+
+  type Query {
+    getAllRepliesByPostId: [Reply]
+    # getReplyById(id: ID!): Reply
+  }
+
+  type Mutation {
+    # createReply(reply: Reply!): Reply
```

Comment:
Can we keep typedefs in the graphql/types folder?

Reply:
yup. moving over now